deno/rest/v8/mod.ts

```diff
@@ -683,3 +683,23 @@ export const OAuth2Routes = {
 
 // Freeze OAuth2 route object
 Object.freeze(OAuth2Routes);
+
+interface DiscordErrorFieldInformation {
+	code: string;
+	message: string;
+}
+
+interface DiscordErrorGroupWrapper {
+	_errors: DiscordError[];
+}
+
+type DiscordError = DiscordErrorGroupWrapper | DiscordErrorFieldInformation | { [k: string]: DiscordError } | string;
```

Comment:
```suggestion
type DiscordError = DiscordErrorGroupWrapper | DiscordErrorFieldInformation | Record<string, DiscordError> | string;
```

Reply:
These threw recursion errors before, don't know if that's the case still